forge.config.js

```diff
@@ -40,6 +41,7 @@ const deletePycFiles = async (directory) => {
 /** @type {import("@electron-forge/shared-types").ForgeConfig} */
 const config = {
     packagerConfig: {
+        asar: false,
```

Comment:
Don't we need this to be `true` for macOS?

Reply:
Do we? We weren't using this with webpack

Reply:
Just checked, everything works with `asar: true`, so we can enable no problem.

Reply:
> Do we?

I remember reading about this in the release notes of forge. I don't actually know though.

Reply:
Okay, [reading the docs](https://www.electronjs.org/docs/latest/tutorial/asar-integrity), I think I remembered incorrectly. My bad.